CMakeLists.txt

```diff
@@ -0,0 +1,307 @@
+## MPAS-Model
+cmake_minimum_required(VERSION 3.12)
+project(MPAS LANGUAGES C Fortran VERSION 8.0 DESCRIPTION "MPAS - Model for Prediction Across Scales")
```

Comment:
Since the latest tag in the history of the `develop` branch (the base for this PR) is `v8.1.0`, I would propose changing the hardwired version number here to 8.1 or 8.1.0.

As @amstokely has mentioned, we'd like to be conservative in our changes at this point simply to avoid breaking the build in MPAS-JEDI.

Reply:
> As an alternative, do you think it would be reasonable to create GitHub issues referring back to outstanding comments not fully resolved to better track resolutions?

I actually addressed this comment [here](https://github.com/MPAS-Dev/MPAS-Model/blob/3f91ab664c655d03c355b91e709f9f5ac60711bd/cmake/Functions/MPAS_Functions.cmake#L26)